client/dashboard/domains/index.tsx

```diff
@@ -1,13 +1,14 @@
 import { useQuery } from '@tanstack/react-query';
-import { Button } from '@wordpress/components';
 import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
 import { __ } from '@wordpress/i18n';
 import { useState } from 'react';
 import { useAuth } from '../app/auth';
 import { domainsQuery } from '../app/queries/domains';
+import { domainsPurchaseRoute } from '../app/router/domains';
```

Comment:
This introduces a circular dependency. If we manage to keep the domains purchase route loading async, then the circular dependency is gone.